### PR TITLE
Added lite member schema

### PR DESCRIPTION
**Pull request by yuraku**

### CODE REVIEW

File: src/main/resources/hazelcast-config-schema.json [hunk 1]
```diff
@@ -1149,6 +1149,16 @@
...
         "class-name"
       ]
+    },
+    "LiteMember": {
+      "additionalProperties": false,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "default": false,
+          "description": "When you want to use a Hazelcast member as a lite member, set this element's \"enabled\" attribute to true in that member's YAML configuration. Lite members do not store data, do not have partitions and are used mainly to execute tasks and register listeners."
```
Comment (erosb) [inline src/main/resources/hazelcast-config-schema.json]: minor comment: if the description says "set this element's \"enabled\" attribute" then this description should be one nesting level higher (so the direct child of `"LiteMember"`).

**Review by erosb — CHANGES_REQUESTED** [review]
One minor comment re. the description added, otherwise it is good, thanks @yuraku , great start!

File: src/main/resources/hazelcast-config-schema.json (hunk 1 above)
Comment (yuraku) [reply src/main/resources/hazelcast-config-schema.json]: make sense. updated

**Review by erosb — APPROVED** [review]
Looks good, thanks @yuraku !

**Pull request merged**